src/dog.sol

```diff
@@ -178,6 +186,8 @@ contract Dog is LibNote {
                 lot: dink,
                 usr: urn
             });
+            
+            vat.suck(address(vow), msg.sender, add(milk.tip, wmul(tab, milk.chip)));
```

Comment:
Just realized this should use `due`, not `tab`. Otherwise it's very confusing to set `chip` correctly, as you have to do an extra correction, potentially, for taking part of `chop` as well.

Reply:
Good point. Do you think I can push this change straight to the `liq-2.0` branch or should I make a PR?

Reply:
Mhm, you'll probably need to tweak the tests too (at least, I hope so, otherwise we need a test to cover this)...better to do a PR then IMO.

Reply:
Yeah good call will do